### ISSUE TITLE
Zombie spawners not working since last update (1.0.4a)

### ISSUE THREAD

**Issue by xAlicatt** [Daveyx0/MultiMob#13]
Me again! Sorry to have to report this, but since the last MM update (1.0.4a) vanilla zombie spawners have ceased functioning. I will be rolling back to the previous version in my pack, but I just wanted you to know. 

One of my players said their zombie spawner wasn't working, so I went to check and it wasn't. I tested in singleplayer...same thing....but all the other vanilla spawners work fine it seems. 

So I just worked backwards from my last pack update and which mob-related mods were updated. Conclusion I have arrived at is that it is MM causing it.

**Comment by AttilaPapp** [comment]
Do you use MM to spawn the zombies? If so, check your spawning restrictions. It happened to me when I wanted to restrict a mob to spawn above a Y level 60 and it caused the spawners not working under that Y level.

**Comment by xAlicatt** [comment]
@AttilaPapp no....I only have MM for Primitive Mobs. There's a bug in the code causing this. It works as expected in version 1.0.4 and they don't work in 1.0.4a

Hopefully this doesn't sound rude, cause I don't mean for it to be, but obviously if I were using MM to spawn the zombies, I would have checked that before making this bug report. :P

**Comment by Freebi85** [comment]
Can confirm this with 1.0.4a

**Comment by AttilaPapp** [comment]
@xAlicatt no probs just thought worth mentioning. Not everybody knows. 😀

**Comment by Daveyx0** [comment]
There is a more underlying problem to this issue. Was not able to get it fixed today. Hopefully have a hotfix tomorrow!

**Comment by xAlicatt** [comment]
All good @Daveyx0 I know it's like 99 bugs....fix one....then 105 bugs xD

**Comment by Daveyx0** [comment]
yes :P Also, fixed in 1.0.5! Sorry for the inconvenience everyone!
